packages/rum-core/src/common/utils.js

```diff
@@ -188,7 +188,7 @@ function getNavigationTimingMarks() {
 function getPaintTimingMarks() {
   var paints = {}
   var perf = window.performance
-  if (perf.getEntriesByType) {
```

Comment:
newbie question: what was this change for? for correctness, so the function call is dependent on it being a function...?

Reply:
Yeah wanted to make sure its a function. Also it's not a `readonly` property, so it can be polyfilled by other JS code on the page.